tests/sentry/replays/unit/test_event_logger.py

```diff
@@ -111,3 +113,169 @@ def test_emit_trace_items_to_eap(producer: mock.MagicMock) -> None:
     assert producer.call_args[0][1].key is None
     assert producer.call_args[0][1].headers == []
     assert isinstance(producer.call_args[0][1].value, bytes)
+
+
+@mock.patch("sentry.replays.usecases.ingest.event_logger.logger")
+@mock.patch(
```

Comment:
Remove this mock and pass a callback as a function argument.  We never mock _if we can avoid it_ and we can _always_ avoid it (when writing new code).